pyHS100/pyHS100.py

```diff
@@ -19,10 +19,16 @@
 import enum
 
 from .protocol import TPLinkSmartHomeProtocol
+from . import schedule
+from . import countdown
+from . import antitheft
 
-_LOGGER = logging.getLogger(__name__)
 
+_LOGGER = logging.getLogger(__name__)
 
+NoScheduleFound = schedule.NoScheduleFound
+NoCountdownFound = countdown.NoCountdownFound
+NoAntiTheftFound = antitheft.NoAntiTheftFound
```

Comment:
we would need to create a separate exception file and also have the SmartPlugException class in it in order to be able to subclass it.